src/repos.ts

```diff
@@ -34,50 +142,32 @@ export async function syncRepoSettings() {
       });
   });
   await Promise.all(ps);
+}
 
-  console.log('Updating master branch protection...');
-
-  // The Go repositories are synchronized with a different git
-  // repository, so direct commits need to be made.
-  const ignoreRepos = [
-    'googleapis/google-cloud-go',
-    'googleapis/google-api-go-client',
-  ];
+const languageConfig: LanguageConfig = {
+  nodejs: {
+    requiredStatusChecks: [
+      'ci/kokoro: Samples test',
+      'ci/kokoro: System test',
+      'ci/kokoro: Windows',
+      'ci/kokoro: docs',
+      'ci/kokoro: lint',
+      'ci/kokoro: node8',
```

Comment:
we'll need to ask each language to populate a similar list? this is great for merge on green CC: @sofisl.

Reply:
As we figure out what we're doing, I suspect we will a.) move this code into a bot and b.) split this specific piece out to a json file that can be used from multiple places.  After this rolls out, I would be comfortable saying we can use merge-on-green for node.js only.  For other languages that opt in here, they get it too.